controllers/istio_ratelimit_cluster_reconciler.go

```diff
@@ -56,6 +56,10 @@ func (r *IstioRateLimitClusterReconciler) Reconcile(ctx context.Context, _ []con
 	if kuadrant == nil {
 		return nil
 	}
+	mtls := false
```

Comment:
```suggestion
	mtls := kuadrant != nil && kuadrant.Spec.MTLS != nil && kuadrant.Spec.MTLS.Enable
```

Reply:
Actually, it could be `mtls := kuadrant.IsMTLSEnabled()` according to https://github.com/Kuadrant/kuadrant-operator/pull/1170/files#r1995757686